backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisWebservice.java

```diff
@@ -24,6 +24,7 @@ public class MolgenisWebservice {
   public static final String MANAGER = "Manager";
   public static final String ROLE = "role";
   public static final String VIEWER = "Viewer";
+  public static final long MAX_REQUEST_SIZE = 10_000_000L;
```

Comment:
Here I think it is because python client has shortcomings. In principle this migth be happening more often for big transactions and then we should considert multipart uploads. Also I can imagine filter based updating to be a way (it is often about deletes). Splitting is not acceptable because of transactionality